torchaudio/backend/common.py

```diff
@@ -60,7 +60,8 @@ def __init__(self,
 _LOAD_DOCSTRING = r"""Loads an audio file from disk into a tensor
 
 Args:
-    filepath: Path to audio file
+    filepath: Path to audio file. This function also handles `pathlib.Path` objects, but is annotated
+             as ``str`` for TorchScript compiler compatibility.
```

Comment:
Actually, the other backend is not TorchScript compatible. So can you evert this?
I think the change to other backends should be separated from this PR. (note: this is not asking you to work on it.)

Reply:
@timlod  Is this accidentally resolved?

If you want to change the cdocsrting of other backends, then can you make it appear not for TorchScript compiler compatibility but for the consistency between "sox_io" backend which cannot be annotated as `pathlib.Path` due to TorchScript compiler compatibility?

Reply:
@mthrok the most recent commit removes that change - am I missing something? I was not planning on changing the other backends (I could do that if you'd like me to, but as you mention above that should go in another PR)

Reply:
Oh, I see you reverted change on `_SAVE_DOCSTRING` but the same has to happen to the other docstrings too.
I think `git checkout b6a61c3f -- torchaudio/backend/common.py` will fix this.

Reply:
Oops - not quite sure what went wrong there, but I fixed it now. Thanks for your patience!